Dockerfile

```diff
@@ -39,10 +42,19 @@ WORKDIR $WORKDIR
 RUN apk --no-cache add \
     mediainfo
 
+# Add user to run the application
+RUN addgroup -S flood
+RUN adduser -S flood -G flood
+RUN mkdir /data
+RUN chown flood:flood /data
```

Comment:
Have you been able to test how it affects current configurations?

Reply:
@mactwister maybe the file are own by root because the docker image was built and the default user was root but if you build it and add another user the file will be own by another user.

Reply:
Yes, so now if you build the docker image on your side using this Dockerfile (with chown changes) then the app will be owned and run as the `flood` user. 👍 

#### For new setups using docker:
New container will run as `flood` user and create the database under `/data` owned by the `flood` user. All good here.

#### For existing setups using docker:

The scenario I am describing is for people already running with Docker and have setup the `/data` folder as a "volume" to persist the DB when restarting/upgrading the docker image. Below is simplified version to how I have my setup:

```docker-compose.yml
# ... rest of you docker compose file
  floodui:
    build: .
    ports:
      - 3000:3000
    volumes:
      - flood_data:/data
```

**So** when re-creating the container with new image, the app will crash with the following error:

```
[Error: EACCES: permission denied, open '/data/server/db/users.db'] {
```

This is because my volume `/data` is persisted over from the previous container, where it was first created (when container was root only) so my version of `/data` is owned by `root:root`.

```bash
julian$ docker-compose run --rm floodui ls -l /data
  total 4
  drwxr-xr-x    3 root     root          4096 Oct  3 08:41 server
```

**What I am saying** is that, if we go through with this change, we'd need to some how inform users what they would need to do to continue working after this upgrade. i.e. update the docs for docker setup to inform the to update the /data ownership manually.

#### What I needed to do using my docker compose setup to continue working:
```bash
julian$ docker-compose run --user=root --rm floodui chown -R flood:flood /data
```